clang/test/CXX/drs/cwg29xx.cpp

```diff
@@ -33,7 +33,7 @@ void *operator new(std::size_t, void *p) { return p; }
 void* operator new[] (std::size_t, void* p) {return p;}
 
 
-namespace cwg2922 { // cwg2922: 20 open 2024-07-10
+namespace cwg2922 { // cwg2922: 20 tentatively ready 2024-07-10
```

Comment:
@cor3ntin can you confirm that the resolution did not change?

Reply:
Yes! Thanks